sched/semaphore/sem_clockwait.c

```diff
@@ -268,6 +260,16 @@ int sem_clockwait(FAR sem_t *sem, clockid_t clockid,
 {
   int ret;
 
+  /* Verify the input parameters and, in case of an error, set
+   * errno appropriately.
+   */
+
+  if (abstime == NULL || sem == NULL)
```

Comment:
```suggestion
  if (sem == NULL || abstime == NULL)
```
just from common sense perspective